src/gened/base.py

```diff
@@ -10,7 +10,7 @@
 from typing import Any
 
 import flask.app
-import pyrage
+import pyrage # type: ignore
```

Comment:
This is causing mypy failures in the CI flow.  If it's needed on your system, maybe open an issue about that and I can look into it, but I don't want to ignore types here in the main branch.